config/webpack/webpack.common.js

```diff
@@ -35,6 +35,11 @@ module.exports = env => {
     __DEV__: isDev && !process.env.LINKED_DEV,
     __config__: JSON.stringify(config),
     __houston_url__: JSON.stringify(houstonUrl),
+    __sentry_dsn__: JSON.stringify(
+      isDev
+        ? process.env.SENTRY_DSN_DEVELOPMENT
+        : process.env.SENTRY_DSN_PRODUCTION,
```

Comment:
Is there a houston PR that sets these environment variables?

Reply:
There's no PR yet but the branch is [sentry-dsn](https://github.com/WildMeOrg/houston/tree/sentry-dsn) and this is the [commit](https://github.com/WildMeOrg/houston/commit/ab20bc9dae716ef27ca107d97271a3e45ce0f805):

```diff
commit ab20bc9dae716ef27ca107d97271a3e45ce0f805 (HEAD -> sentry-dsn, origin/sentry-dsn)
Author: karen chan <karen@karen-chan.com>
Date:   Wed Jun 1 13:48:55 2022 +0100

    Add sentry environment variables to dev-frontend docker-compose service
    
    We need to give dev-frontend the same environment variables as back end
    so both are using the same sentry dsn.

diff --git a/docker-compose.codex.yml b/docker-compose.codex.yml
index 26d7ab8e..798691a0 100644
--- a/docker-compose.codex.yml
+++ b/docker-compose.codex.yml
@@ -411,6 +411,8 @@ services:
       - intranet
     environment:
       HOUSTON_APP_CONTEXT: codex
+      SENTRY_DSN_DEVELOPMENT: "${SENTRY_DSN_DEVELOPMENT}"
+      SENTRY_DSN_PRODUCTION: "${SENTRY_DSN_PRODUCTION}"
       # See port served by 'www' component (i.e. the reverse proxy)
       HOST: "0.0.0.0"
       PORT: "84"
diff --git a/docker-compose.mws.yml b/docker-compose.mws.yml
index 99afb0c8..8820fb88 100644
--- a/docker-compose.mws.yml
+++ b/docker-compose.mws.yml
@@ -373,6 +373,8 @@ services:
       - intranet
     environment:
       HOUSTON_APP_CONTEXT: mws
+      SENTRY_DSN_DEVELOPMENT: "${SENTRY_DSN_DEVELOPMENT}"
+      SENTRY_DSN_PRODUCTION: "${SENTRY_DSN_PRODUCTION}"
       # See port served by 'www' component (i.e. the reverse proxy)
       HOST: "0.0.0.0"
       PORT: "84"
```